realtimeplotter/plotrecorder.py

```diff
@@ -40,7 +40,8 @@ def record(self, var_name, var_value):
         :param var_name: Name of variable to record
         :param var_value: Value of variable to record
         """
-        assert var_value != SENTINEL, "You cannot record a value {} since this conflicts with the internal SENTINEL string"
+        assert not isinstance(var_value, type(SENTINEL)) or var_value != SENTINEL,\
```

Comment:
Well, I don't see why we should disallow the value from being a string (type of the sentinel) here